scripts/intfutil

```diff
@@ -262,16 +280,18 @@ def po_speed_dict(po_int_dict, appl_db):
                     # If no speed was returned, append None without format
                     po_list.append(None)
                 else:
-                    interface_speed = '{}G'.format(interface_speed[:-3])
+                    optics_type = state_db_port_optics_get(appl_db, value[0], PORT_OPTICS_TYPE)
+                    interface_speed = appl_db_port_speed_parse(interface_speed, optics_type)
                     po_list.append(interface_speed)
             elif len(value) > 1:
                 for intf in value:
                     temp_speed = appl_db.get(appl_db.APPL_DB, "PORT_TABLE:" + intf, "speed")
+                    optics_type = state_db_port_optics_get(appl_db, intf, PORT_OPTICS_TYPE)
                     temp_speed = int(temp_speed) if temp_speed else 0
                     agg_speed_list.append(temp_speed)
                     interface_speed = sum(agg_speed_list)
                     interface_speed = str(interface_speed)
-                    interface_speed = '{}G'.format(interface_speed[:-3])
+                    interface_speed = appl_db_port_speed_parse(interface_speed, optics_type)
```

Comment:
what happens if a po consists of a 1000M RJ45 and a 1G SFP ports? Is that a legal scenario?
Do we need to add test case for po intf?

Reply:
it's illegal,  teamd should prevent this case